src/components/icon/icon.stories.mdx

```diff
@@ -0,0 +1,88 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Overview, Regular } from './icon.stories';
+import { Icon, IconSize } from './icon'; import Callout from '../callout/callout';
+
+<Meta
+  title="Components/Icon"
+  component={{ Icon }}/>
+
+# Icon
+The default size is set to `normal` which equals `24px`. You can set a size of `inherit` to grow with the text (where it's size 150% of your text size). 
+
+<Canvas withSource="open">
+  <Story story={Regular}/>
+</Canvas>
+
+<ArgsTable of={Icon} />
+
+## Overview
+Find here the all available icons and their name.
+
+<Story name="Overview">
+  <Overview/>
+</Story>
+
+
+## Usage
+The icon can be used in various situations. Here some examples with buttons and inside a text.
+<div style={{margin: '20px 0'}}>
+  <Callout illustration={'scientist_042'}>
+    This is not a component that magically works everywhere.
+    You want to adjust the icons in particular situations to fit the given design expectations.
+  </Callout>
+</div>
+
+### With Buttons
+
+<Canvas>
+  <Story name="With Buttons">
+    <div>
+      <button>
+        <Icon show={'theme_switch'} size={IconSize.NORMAL} />
+      </button>
+      <button>
+        <Icon show={'burger_nav'} size={IconSize.NORMAL} /> Menu
```

Comment:
As far as I understood, `IconSize.NORMAL` is the default. Why would we want to explicitly specify this here, then? It makes the code for this story more confusing then it would need to be in my opinion.